lib/views/shared/receive.dart

```diff
@@ -133,13 +132,40 @@ class ReceiveSharedState extends ChangeNotifier {
       if (error is ClientError) {
```

Comment:
In case error is not ClientError, we will still throw error, but in that case message will not be display in See Details screen. This leads, that some technical or not mapped error will be displayed unhidden.